ui/src/js/viewModels/orgAdmin.js

```diff
@@ -7,436 +7,444 @@
                                 }).then(function () {
                                     self.userOrgLoaded(true);
                                 })
-                            );
-                        };
+                        );
+                    };
 
-                        self.getUserOrgData = function (orgId) {
-                                //GET /rest/organizations - REST
-                                return $.when(restClient.doGet('/rest/organizations/' + orgId + '/user_organizations')
-                                    .then(
-                                        success = function (response) {
-                                            self.userOrgValues(response.user_organizations);
-                                            var userOrgs = self.userOrgValues().filter(function(user){
-                                                user.adminTable = [];
-                                                if (user.admin === "Y") {
-                                                    user.adminTable = ['checked'];
-                                                }
-
-                                            });
-                                            var sortCriteria = {key: 'name', direction: 'ascending'};
-                                            var arrayDataSource = new oj.ArrayTableDataSource(self.userOrgValues(), {idAttribute: 'user_id'});
-                                            arrayDataSource.sort(sortCriteria);
-                                            self.userOrgDataProvider(new oj.PagingTableDataSource(arrayDataSource));
-                                            self.userOrgLoaded(true);
-                                            self.userOrgValid(true);
-
-                                        },
-                                        error = function (response) {
-                                            console.log("User organizations not loaded");
-                                            self.userOrgValid(false);
+                    self.getUserOrgData = function (orgId) {
+                        //GET /rest/organizations - REST
+                        return $.when(restClient.doGet('/rest/organizations/' + orgId + '/user_organizations')
+                            .then(
+                                success = function (response) {
+                                    self.userOrgValues(response.user_organizations);
+                                    var userOrgs = self.userOrgValues().filter(function (user) {
+                                        user.adminTable = [];
+                                        if (user.admin === "Y") {
+                                            user.adminTable = ['checked'];
+                                        }
 
-                                        })
+                                    });
+                                    var promises = [];
```

Comment:
This was recently removed. Why was it added back?

Reply:
Merge conflicts, we'll refactor this.
Why was it removed?

Reply:
It was there in the past to get the full user details to show on this screen, but now we navigate to the users screen instead, so we don't need the full user details here. Yes, we shoud re-remove it.

Reply:
I tried removing the var promises = []; from my code, but it gave an error and the org admin page did not load.

> jQuery.Deferred exception: promises is not defined ReferenceError: promises is not defined
    at Object.<anonymous> (http://localhost/ui/web/js/viewModels/orgAdmin.js:386:41)
    at Function.each (http://localhost/ui/web/js/libs/jquery/jquery-3.5.1.js:381:19)
    at success (http://localhost/ui/web/js/viewModels/orgAdmin.js:385:39) undefined

that is the error I got in the console tab.
In the network tab, it shows that it did successfully load the user details.